cmd/kube-scheduler/app/options/plumbing.go

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
Is there any benefit of creating this new file and not have it as part of options/options.go?

Reply:
no, there isn't. merged them.